docs/pages/server-access/rbac.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: Server Access Controls
+description: Role-based access control (RBAC) for Teleport Server Access.
+---
+
+# Server Access Role-Based Access Control
+
+Role-based access control (or RBAC, for short) allows administrators to set up
+granular access policies for servers connected to Teleport.
+
+An example of a policy could be, *"server administrators have access to
+everything, QA team and engineers have full access to staging servers, and
+engineers can gain temporary access to the production server in case of
+emergency"*.
+
+For a more general description of Teleport roles and examples see [RBAC](../access-controls/introduction.mdx), as
```

Comment:
I think we should flip this sentence around so "this section focuses on configuring RBAC for Server Access" comes first. That statement is the more important part of this paragraph. Also, I think we should say "guide" instead of "section."